edweiss-app/app/(app)/quiz/createQuizPage.tsx

```diff
@@ -469,6 +461,17 @@ export const EditSecondModal: ReactComponent<{ modalRef: RefObject<BottomSheetMo
 export const MCQFields: ReactComponent<{ addToExerciseList?: (exercise: Quizzes.Exercise) => void, editExercise?: (exercise: Quizzes.Exercise, index: number) => void, modalRef?: RefObject<BottomSheetModalMethods>, previousPropositions?: [string, boolean][], previousQuestion?: string, index?: number }> = (props) => {
 	const [question, setQuestion] = useState<string>(props.previousQuestion == undefined ? "" : props.previousQuestion);
 	const [propositions, setPropositions] = useState<[string, boolean][]>(props.previousPropositions == undefined ? [] : props.previousPropositions);
+	const [questionError, setQuestionError] = useState<string | undefined>(undefined);
+
+	function handleQuestionInputError(text: string) {
+		if (text.length == 0) {
+			setQuestionError("Please write a question")
```

Comment:
This is hard coded English, please use the `t` function.

Reply:
Language support will come next week.